docs/book/src/reference/good-practices.md

```diff
@@ -75,10 +75,13 @@ Therefore, you can check an example of Status Conditional usage by looking at it
 
 </aside>
 
-[docs]: ./cronjob-tutorial/gvks.html
+[docs]: /cronjob-tutorial/gvks.html
 [operator-pattern]: https://kubernetes.io/docs/concepts/extend-kubernetes/operator/
 [controllers]: https://kubernetes.io/docs/concepts/architecture/controller/
 [controller-runtime-topic]: https://github.com/kubernetes-sigs/controller-runtime/blob/main/FAQ.md#q-how-do-i-have-different-logic-in-my-reconciler-for-different-types-of-events-eg-create-update-delete
 [controller-runtime]: https://github.com/kubernetes-sigs/controller-runtime
-[deploy-image]: ./plugins/deploy-image-plugin-v1-alpha.md
+[deploy-image]: /plugins/deploy-image-plugin-v1-alpha.md
 [controller-runtime-manager]: https://github.com/kubernetes-sigs/controller-runtime/blob/304027bcbe4b3f6d582180aec5759eb4db3f17fd/pkg/manager/manager.go#L53
+[k8s-api-conventions]: https://kubernetes.io/docs/reference/using-api/api-concepts/
```

Comment:
Hello there👋 I would recommend to use this link https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md for referring to K8s API conventions. The main difference here is that [API Concepts](https://kubernetes.io/docs/reference/using-api/api-concepts/) is rather suitable for using the API instead of standards and conventions from an architectural point of view. What do you think?